Mathlib/Algebra/Module/Submodule/Defs.lean

```diff
@@ -165,7 +163,10 @@ variable {p q : Submodule R M}
 variable {r : R} {x y : M}
 variable (p)
 
--- Porting note: removing `@[simp]` since it can already be proven
+-- In Lean 3, `dsimp` would use theorems proved by `Iff.rfl`.
+-- If that were still the case, this would useful as a `@[simp]` lemma,
+-- despite the fact that it is provable by `simp` (by not `dsimp`).
+@[simp, nolint simpNF] -- See https://github.com/leanprover-community/mathlib4/issues/10675
```

Comment:
You split this into a dependent PR, right? Then I won't review this in detail/would like another opinion on this one.

Reply:
No, Kim made a separate PR but that didn't include this instance.